Eclipse hawkBit is now licensed under EPL-2.0

shogo82148 opened the pull request:
https://github.com/eclipse/hawkbit/commit/9c86729a68af97a23f56e4fbe9be38b863895981

shogo82148 committed:
Eclipse hawkBit is now licensed under EPL-2.0

https://github.com/eclipse/hawkbit/commit/9c86729a68af97a23f56e4fbe9be38b863895981

shogo82148 committed:
add ClojureScript for EPL-1.0

shogo82148 committed:
Eclipse SmartHome is archived

shogo82148 committed:
openHAB and SUMO changed their default branch

mlinksva commented:
Thanks for noticing. We don't need another 2.0 example, but do need to replace hawkbit as a 1.0 example. https://github.com/clojure/clojurescript/blob/master/LICENSE seems like an obvious pick.

mlinksva committed:
Skip foundation name

I realize we weren't consistent before, taking opportunity to be so now

shogo82148 commented:
> We don't need another 2.0 example

Right. We don't need 4 examples.
But there are other examples that should be removed.
I removed [Eclipse SmartHome](https://github.com/eclipse-archived/smarthome) because it is archived.

> https://github.com/clojure/clojurescript/blob/master/LICENSE seems like an obvious pick.

I agree. I added ClojureScript on https://github.com/github/choosealicense.com/pull/1153/commits/d9f9de4c6368f7d2f59faa26ef55b96bf722c6fe.

mlinksva committed:
replace skater with graaljs

mlinksva commented:
Oh I did not see #1154 @shogo82148 glad we chose the same project.
